### DISCUSSION TITLE
regolith-look-solarized-dark on Ubuntu 23.10

### DISCUSSION THREAD

**Discussion by ChunkyGrumbler** [regolith-linux/regolith-desktop#944] [General]
On Ubuntu 23.10, clean install:
```
sudo apt install regolith-look-solarized-dark
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 solarc-theme : Depends: gnome-themes-standard but it is not installable
E: Unable to correct problems, you have held broken packages.
```

`gnome-themes-standard` was a transitional package, and it looks like it was removed at Lunar. `solarc-theme` should probably depend on `gnome-themes-extra`

If someone could point me at the right repo, I could have a look at doing a PR.

Cheers,
    Andy.

**Comment by kgilmer** [comment]
Hi @ChunkyGrumbler!

  The Regolith build system defines all package sources (git url & ref) here: https://github.com/regolith-linux/voulage/tree/main/stage.  In particular to this issue, the `unstable` package reference for `solarc-theme`: https://github.com/regolith-linux/voulage/blob/main/stage/unstable/package-model.json#L39.  The package that has the problem: https://github.com/regolith-linux/solarc-theme.  If you make a PR against that I can take care of the build stuff.  LMK if there are any details I've missed!  :rocket: 

TIA, ken

**Reply by ChunkyGrumbler** [reply]
Hi @kgilmer,

Thanks, I've done the PR.

I was unable to build this change down to a .deb on my machine. Are there some instructions for building individual packages down to `dpkg` installable images?

Thanks,
    Andy.

**Reply by kgilmer** [reply]
Yes, although you wouldn't be able to sign them with the Regolith key.  Here are some links:

https://blog.packagecloud.io/buildling-debian-packages-with-debuild/
https://metebalci.com/blog/a-minimum-complete-example-of-debian-packaging-and-launchpad-ppa-hellodeb/

**Comment by kgilmer** [comment]
Verified this is fixed in Regolith 3.1.  Verified on Ubuntu Mantic
